src/structures/Webhook.js

```diff
@@ -226,6 +226,18 @@ class Webhook {
     return this;
   }
 
+  /**
+   * Gets a message that was sent by this webhook.
+   * @param {Snowflake} message The ID of the message to fetch
```

Comment:
```suggestion
   * @param {MessageResolvable} message The ID of the message to fetch
```

Mirror this from the similar methods (and possibly add `@original` to the typedef for MessageResolvable, that I'm not sure on though).

Reply:
I did consider using a `MessageResolvable` type at first but [`MessageManager#fetch()`](https://github.com/izexi/discord.js/blob/feat/Webhook%23fetchMessage()/typings/index.d.ts#L2043) accepts a `Snowflake` as the parameter so I went ahead and used that here as well to be consistent with that. Which similar methods are you referring to here?

Reply:
I misread actually, but I was referencing editMessage and deleteMessage. You are directly passing the message token here so MessageResolvable is actually incorrect 🤦